src/main/java/com/conveyal/gtfs/loader/conditions/ConditionalRequirement.java

```diff
@@ -0,0 +1,41 @@
+package com.conveyal.gtfs.loader.conditions;
+
+import com.conveyal.gtfs.error.NewGTFSError;
+import com.conveyal.gtfs.loader.Field;
+import com.conveyal.gtfs.loader.LineContext;
+import com.google.common.collect.HashMultimap;
+
+import java.util.Set;
+
+/**
+ * These are the requirements that are checked inline with {@link ConditionalCheckType} to determine if the required
+ * conditions set forth for certain fields in the GTFS spec have been met. These requirements are applied directly to
+ * their "reference fields" with the help of {@link Field#requireConditions}.
+ */
+public abstract class ConditionalRequirement {
+    /** The type of check to be performed on a reference field. A reference field value is used to determine which check
+     * (e.g., {@link AgencyHasMultipleRowsCheck#check}) should be applied to the field. */
+    protected ConditionalCheckType referenceFieldCheck;
+    /** The minimum reference field value if a range check is being performed. */
+    protected int minReferenceValue;
+    /** The maximum reference field value if a range check is being performed. */
+    protected int maxReferenceValue;
```

Comment:
These are only ever used in the `FieldInRangeCheck` ConditionalRequirement. It's probably safe to move these variables to that class.

Reply:
I ended up implementing this in a proposed refactor: #320